code/modules/uplink/uplink_items.dm

```diff
@@ -1233,6 +1233,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	surplus = 0
 	restricted_roles = list("Cook", "Botanist", "Clown", "Mime")
 
+/datum/uplink_item/role_restricted/poisoned_apple
+	name = "Poisoned Apple"
+	desc = "A deadly apple. Can't be reproduced from seeds. Nanotrasen Botanists are all the same, all flash and no fury."
+	item = /obj/item/reagent_containers/food/snacks/grown/apple/poisoned
+	cost = 4
+	surplus = 0
```

Comment:
Is this a T/F?

Reply:
All other job specific traitor items are not accessible in the surplus crate, and that's what setting it to 0 does.

Reply:
oh it's a probability smh!